### ISSUE TITLE
[Feature Request]: Reopen an issue if it corresponds to unresolved TODOs in the codebase.

### ISSUE THREAD

**Issue by seanlip** [oppia/oppiabot#316]
**Is your feature request related to a problem? Please describe.**

The Oppia Android repo has a nice feature where Oppiabot reopens an issue if there are unresolved TODOs left in the codebase, e.g. https://github.com/oppia/oppia-android/issues/3016#issuecomment-939154165

We should do this for Oppia Web as well.

**Describe the solution you'd like**
When an issue is closed, Oppiabot should scan the codebase for TODOs corresponding to that issue, and reopen the issue if such TODOs are found.

**Describe alternatives you've considered**
N/A

**Additional context**
We can probably just repurpose what's already written for Oppia Android.

**Comment by seanlip** [comment]
See also https://github.com/oppia/oppia-android/issues/4930 which suggests a *better* system than Oppia Android's status quo for handling this (i.e. triggering the TodoCheck workflow on PR creation).

**Comment by BenHenning** [comment]
FYI: Oppia Android doesn't actually use Oppiabot for this functionality, it just reopens issues using the gh CLI tool via a GitHub Actions workflow: https://github.com/oppia/oppia-android/blob/4a14b415a4fba74a415e9fa4427d37a585a55569/.github/workflows/issue_checks.yml#L35.

**Comment by vojtechjelinek** [comment]
@seanlip This is more complicated than expected as the GH Action workflow that Ben suggests uses Bazel script which would need to be rewritten to Python in order to be used in our web repo. I think this would take 1 or 2 persondays to implement.

**Comment by jnvtnguyen** [comment]
@seanlip This should be made using github workflows and not oppiabot right?

**Comment by seanlip** [comment]
Yup, github workflows. Thanks! (I updated the description and am transferring this to oppia/oppia.)
